mentors/javascript/a.json

```diff
@@ -54,5 +54,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I'm a sofware architect & engineer and I love to share knowledge, because I think that everybody should be able to learn how to develop software."
+  },
+  {
+    "github_username": "archanid",
+    "name": "Archana Sriram",
+    "link_text": "github",
+    "link_url": "https://github.com/archanid",
+    "avatar_url": "https://avatars0.githubusercontent.com/u/13165182?s=460&v=4",
```

Comment:
If this is your regular GitHub avatar, you can use `null` so that it updates as you update your GitHub avatar.

Reply:
Oh sweet!